{{cookiecutter.project_slug}}/clients/web/react/tests/e2e/specs/sign-up.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { test, expect } from '@playwright/test'
+
+const PASSWORD = 'PASSWORD'
+
+test('Login workflow', async ({ page }) => {
+  await page.goto('/sign-up')
+  await page.getByTestId('first-name').fill('playwright')
+  await page.getByTestId('last-name').fill('e2e test')
+  await page
+    .getByTestId('email')
+    .fill(`playwright${Math.floor(Math.random() * 1000)}@thinknimble.com`)
```

Comment:
I suggest using a UUID or a larger number here to greatly reduce the chance of a collision.

Reply:
yeah you are right, I replaced it for a timestamp instead as its value is unique

Reply:
There is still a chance of collision when browser tests are run in parallel, but that should be very unlikely, so this is a lot better. Thanks!